libevmasm/Assembly.cpp

```diff
@@ -456,6 +465,14 @@ map<u256, u256> Assembly::optimiseInternal(OptimiserSettings _settings)
 		}
 	}
 
+	/// Run again after Deduplicate or CSE
```

Comment:
I've seen some cases where dedup/cse left unused tags. Admittedly they could be improved to not do that.